### PR TITLE
Hamburger features

### PR DESCRIPTION
**Pull request by fmshot2**
This repository is my microverse portfolio project. In this branch, I added a dynamic hamburger menu using javascript.
Here are the steps I took:
a. Added a click event to the existing hamburger icon to generate a dynamic active class on the menu items.
b. Styled the new active classes generated to conform with the figma UI provided.
c. Added the corresponding href values so as to link to the right sections of the page.

Thank you.

### CODE REVIEW

File: navbar.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+const hamburger = document.querySelector('#clicktoggle');
+hamburger.addEventListener('click', () => {
```
Comment (tufoinnkuo10) [inline navbar.js:2]: - [ ] Great job so far. Kindly ensure that on click on any of the nav links the popup menu disappears

File: navbar.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  const hamburger = document.querySelector('.hamburger');
+  const navigate = document.querySelector('.navigate');
+  const headerItems = document.querySelector('.header-items');
```
Comment (tufoinnkuo10) [inline navbar.js:5]: - [ ] please ensure that a user is not able to scroll through your portfolio when the menu has not yet been closed or a link clicked. 
Screenshot (508)_LI

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @fmshot2 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ Mobile menu hamburger works as expected
✅ Menu close icon works perfectly well

Required Changes ♻️
Check the comments under the review
Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by GzimAsani — COMMENTED** [review]
# STATUS: APPROVED :100:         
## There is nothing else to say than it's time to merge... :partying_face: 
Cheers and Happy coding! :clap:  :computer:  :wine_glass: 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


## Optional suggestions
-N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- NA

### Please be reminded to not open a new pull request after you get a review, only use the same one :thought_balloon: 


 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**